frontend/src/components/cards/course-card.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+import { ratingToStars } from "~/lib/ratings-to-stars";
+import { type AvgMetrics, type CourseProfSchema } from "~/lib/zod-schema";
+import { Card, CardContent, CardHeader } from "../ui/card";
+
+export default async function CourseCard(props: {
+  courseId: number;
+  title: string;
+  professors: CourseProfSchema;
+  avgMetrics: AvgMetrics;
+}) {
+  console.log(props.professors);
+  return (
+    <Link href={`/courses/${props.courseId}`}>
```

Comment:
See below comment, if you decide to change it in this ticket, feel free to keep `courseId` in the passed props for usage like this, but we'll want to add `courseCode` for the actual display.